.github/workflows/markdownlint-problem-matcher.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Is this going to do anything in dotnet/runtime repo? We do not have github pipelines enable in this repo and we do not plan them enabled in foreseeable future (it is the long-term eventual goal).

Reply:
AFAIK, the problem matcher points to places of markdownlint errors in the "Files changed" tab in the PR.

Reply:
The markdownlint.yml file is the one responsible for running a GitHub action.

Reply:
Since we have Github actions disabled, does this manifest in the repo experience in any way?

Reply:
@jkotas I don't think it can be useful if GitHub actions are disabled.